.changeset/hip-eagles-sleep.md

```diff
@@ -0,0 +1,5 @@
+---
+"@changesets/action": patch
+---
+
+Provider steps.changesets.output.shouldPublish to github workflow
```

Comment:
In a way this is just a docs PR so let's not add a changeset here.
```suggestion
---
---
```

Reply:
Removed. This was left over from when I was making more drastic changes. Thanks.